### ISSUE TITLE
Accessibility bug - declarative table filter is not announcing empty results

### ISSUE THREAD

**Issue by bharathpalaksha** [microsoft/azuredatastudio#25446]
- Azure Data Studio Version: 1.47.0
- OS Version: windows 11, mac and linux
- Extension: Azure Cosmos DB Migration for MongoDB

This accessibility bug is on the extension but we use Azure data studio components, we have declarative table but when we call setfilter() for empty results, it doesn't announce that there are no table values. 

Steps to Reproduce:

https://github.com/microsoft/azuredatastudio/assets/342603/c9de0c7c-eda5-4199-b8c4-393e5302cd26

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by Samhitha66** [comment]
#A11yMAS; #SH_Azure Data Studio_Nov2023_Linux_Dec2023; #SQL Azure Data Studio;
